packages/astro/src/runtime/server/render/csp.ts

```diff
@@ -24,7 +24,18 @@ export function renderCspContent(result: SSRResult): string {
 			return `${type} ${value}`;
 		})
 		.join(';');
-	const scriptSrc = `style-src 'self' ${Array.from(finalStyleHashes).join(' ')};`;
-	const styleSrc = `script-src 'self' ${Array.from(finalScriptHashes).join(' ')};`;
+
+	let scriptResources = "'self'";
+	if (result.scriptResources.length > 0) {
+		scriptResources = result.scriptResources.map((r) => `'${r}'`).join(' ');
```

Comment:
Are there scenarios where this could need escaping?

Reply:
Not that I'm aware of. URLs don't have apostrophes, and the other resources are well-known